.gitignore

```diff
@@ -140,4 +140,5 @@ fighthealthinsurance/static/js/dist
 
 # Ignore external storage
 external_storage
-external_data
\ No newline at end of file
+external_data
+.DS_Store
```

Comment:
Good call, you can also add this to your systemwide ignore but given the prevalence of mac +1